src/style.css

```diff
@@ -1 +1,84 @@
-/* Styles go here */
+* {
+  font-family: "Roboto", sans-serif;
+  box-sizing: border-box;
```

Comment:
Only `box-sizing` allowed for this selector. Move other to html or body selector